Heisenberg4269/LCA.cpp

```diff
@@ -0,0 +1,83 @@
+class Solution
+{
+public:
+    vector < vector<int> > tree;
+    vector <int> depth;
+    vector < vector<int> > ancestor;
+
+
+    Solution(vector < vector<int> > fn_tree)
+    {
+        tree = fn_tree;
+        int num_nodes = tree.size();
```

Comment:
This is okay for now, but ideally must be size_t or unsigned int.

Reply:
`const int num_nodes` because we don't plan on changing the variable anywhere in the code.

Reply:
In this case, is it better to define num_nodes as a member variable without const or inside the function as const?